src/VisualStudio/Core/Test/Microsoft.VisualStudio.LanguageServices.UnitTests.vbproj

```diff
@@ -7,6 +7,11 @@
     <UseWpf>true</UseWpf>
     <RootNamespace></RootNamespace>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="UnifiedSettings\**" />
+    <EmbeddedResource Remove="UnifiedSettings\**" />
+    <Page Remove="UnifiedSettings\**" />
+  </ItemGroup>
```

Comment:
Is this necessary? It seems like these all reference a location which does not exist.

Reply:
It should be removed ; ) VS Auto create this for me